change map orientation

aparlato opened the pull request:
Closes https://github.com/stamen/maperture/issues/94

This PR changes the orientation of maps in the mirror layout to use a column approach with exceptions at 2 and 6 maps to keep maps either square or in portrait orientation (as opposed to landscape).

### QA
- [ ] In mirror layout, confirm any number of maps looks good and tends toward a portrait layout

ebrelsford commented on src/components/MapsMirrorLayout.svelte:
```diff
@@ -39,31 +54,63 @@
+
+  .map-container-column:last-of-type {
+    border-bottom: none;
+  }
```
That's fair, I can see how explicit classes is nice in some ways.

When I see a selector like `.map-container-column` I expect it to be a column, so it's confusing when it's not. Renaming it `.column-map-container` or `.mirror-column-map-container` or similar feels a little clearer to me. At that point I wonder if it's worth using more complex selectors (say, `.mirror-column-orientation .map-container`).

What do you think about either renaming these or using nested selectors?

aparlato commented:
@ebrelsford good call. I just went ahead and simplified this with nested selectors like you suggest. I also simplified the way we handle borders. This has the side effect that borders between maps are _slightly_ (1px) thicker since the border property doesn't appear to let you draw a border at 0.5px. I tried to mitigate this thickness by using a slightly less harsh color than full black. Let me know if this works for you!
